### PR TITLE
fix(byods-docs): added readme file

### PR DESCRIPTION
**Pull request by Parimala032**
# COMPLETES <[SPARK-549031](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-549031)>

## This pull request addresses

This Pull request is updated with the main README.md file for the byods 

## by making the following changes

Added the file in package/byods 

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios were tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [ ] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [ ] I discussed changes with code owners prior to submitting this pull request

- [ ] I have not skipped any automated checks
- [ ] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/byods/README.md [hunk 1]
```diff
@@ -1,3 +1,63 @@
...
+
+## Testing
+
+```bash
+ yarn workspace @webex/byods run test
+```
+
+## Samples 
+```bash
+  yarn run samples:serve
```
Comment (adhmenon) [inline packages/byods/README.md:42]:
```suggestion
  yarn workspace @webex/byods-demo-server dev:hot
```
Will need to get the demo server changes in order for this to work though.

File: packages/byods/README.md [hunk 2]
```diff
@@ -1,3 +1,63 @@
...
+yarn install
+```
+
+## Building
+
+If your project needs some additional steps for the developer to build the
+project after some code changes, state them here:
+
+```bash
+yarn workspaces foreach --parallel --verbose run build:src
```
Comment (adhmenon) [inline packages/byods/README.md:29]: We can remove this.. no need to run for each package.

File: packages/byods/README.md [hunk 3]
```diff
@@ -1,3 +1,63 @@
...
+
+yarn build:src
```
Comment (adhmenon) [inline packages/byods/README.md:31]:
```suggestion
yarn workspace @webex/byods run build:src
```

File: packages/byods/README.md (hunk 2 above)
Comment (adhmenon) [inline packages/byods/README.md:29]: We can remove this.... no need to run all.

**Review by adhmenon — APPROVED** [review]
LGTM.

File: packages/byods/README.md [hunk 4]
```diff
@@ -1,3 +1,61 @@
...
+- [Samples](#samples) 
+- [Consuming SDK](#consuming-sdk)
+  - [NPM](#npm)
+  - [CDN](#cdn)
+---
 
-This is the readme placeholder for the Bring Your own Data Source (BYoDS) SDK.
\ No newline at end of file
+## Getting Started
+With the BYoDS SDK, you can easily integrate device management capabilities into your Webex applications, allowing users to bring their own devices seamlessly into the Webex ecosystem.
```
Comment (bhabalan) [inline packages/byods/README.md]: Can you please check this description? This is not aligned with what the BYoDS SDK does.

File: packages/byods/README.md [hunk 5]
```diff
@@ -1,3 +1,61 @@
...
+(or)
+
+```javascript
+  yarn add @webex/byods
+```
+
+```javascript
+  import BYoDS from '@webex/byods';
+```
+# CDN
```
Comment (bhabalan) [inline packages/byods/README.md]: We won't be doing CDN deployment as it's a server side SDK

**Review by bhabalan — CHANGES_REQUESTED** [review]

**Review by bhabalan — APPROVED** [review]

**Pull request merged**